storage/backup/backup-cli/src/backup_types/epoch_ending/backup.rs

```diff
@@ -57,8 +57,7 @@ impl EpochEndingBackupController {
     pub async fn run(self) -> Result<FileHandle> {
         info!(
             "Epoch ending backup started, starting from epoch {}, until epoch {} (excluded).",
-            start_epoch = self.start_epoch,
-            end_epoch = self.end_epoch,
+            self.start_epoch, self.end_epoch,
```

Comment:
So, this looks like an issue with the structured logging setup.

This should probably be before the message
```
start_epoch = self.start_epoch,
end_epoch = self.end_epoch,
```

and this should be after
```
self.start_epoch, self.end_epoch
```
@msmouse is this your intention?

Reply:
this shouldn't be changed, this adjusts how the (un)structured logging works.

Reply:
> this shouldn't be changed, this adjusts how the (un)structured logging works.

Well, I changed it back